README.md

```diff
@@ -1,16 +1,20 @@
 # `nixfmt`
 
-`nixfmt` is a formatter for Nix code, intended to easily apply a uniform style.
-
-You are encouraged to test this out on your code and submit any undesirable formatting you find as an issue
+`nixfmt` is the official formatter for Nix language code, intended to easily apply a uniform style.
 
 ![Build Status](https://github.com/NixOS/nixfmt/actions/workflows/main.yml/badge.svg?branch=master)
 
 ## State
 
-`nixfmt` will form the basis for the initial official standard Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+`nixfmt` was originally developed by [Serokell](https://github.com/serokell).
+It was used as the basis for the official standardised Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+
+The official standard differs considerably from the original implementation.
+Be aware of this if you track the [`master`](https://github.com/NixOS/nixfmt/tree/master) branch.
```

Comment:
This sentence is ambiguous; it could either mean "be aware the `master` branch is outdated, following the old style, while development happens on another branch" **or** it could mean the opposite: "be aware the `master` branch is unstable, making frequent breaking changes as we work towards the upcoming stable release implementing the new official standard".

I believe the latter is the correct interpretation, but I'm unsure how to phrase it clearly. Perhaps something like:

```suggestion
Be aware the [`master`](https://github.com/NixOS/nixfmt/tree/master) branch is now implementing the new official standard, **not** the original style used by the [latest release](https://github.com/NixOS/nixfmt/releases/tag/v0.6.0).
```

Reply:
```suggestion
Be aware the [`master`](https://github.com/NixOS/nixfmt/tree/master) branch implements the new official standard, **not** the original style from the [last release made by Serokell](https://github.com/NixOS/nixfmt/releases/tag/v0.6.0).
```

But that repeats half of the other sentences, so I'd add that bit in coordination with the other statements.